lib/database/adapters/index.js

```diff
@@ -108,6 +110,8 @@ runAdapter.lhcPeriodAdapter = lhcPeriodAdapter;
 
 lhcPeriodStatisticsAdapter.lhcPeriodAdapter = lhcPeriodAdapter;
 
+qcFlagTypeAdapter.userAdapter = userAdapter;
```

Comment:
If you look carefully, you'll see that adapters are sorted alphabetically
(I know I did not noticed for the lhcPeriodStatistics adapter)